argus/action_manager/windows.py

```diff
@@ -176,12 +177,37 @@ def git_clone(self, repo_url, location):
         :param location:
             Specific location on the instance.
         """
-        LOG.info("Cloning from %s to %s", repo_url, location)
-        cmd = "git clone {} {}".format(repo_url, location)
-        self._client.run_command_with_retry(cmd,
-                                            count=util.RETRY_COUNT,
-                                            delay=util.RETRY_DELAY,
-                                            command_type=util.CMD)
+        LOG.info('Trying to clone from %r to "%s".', repo_url, location)
+        cmd = 'git clone {repo_url} "{location}"'.format(
+            repo_url=repo_url, location=location)
+
+        for _ in range(util.RETRY_COUNT):
```

Comment:
Use a `while` loop in order to detect the last iteration, doing so you can avoid an useless sleep.